src/ontology/mondo-edit.obo

```diff
@@ -246253,8 +246208,8 @@ xref: OMIM:610156 {source="MONDO:equivalentTo", source="Orphanet:75858", source=
 xref: Orphanet:75858 {source="OMIM:610156", source="MONDO:equivalentTo"}
 xref: SCTID:715628009 {source="MONDO:equivalentTo"}
 xref: UMLS:C1857802 {source="OMIM:610156", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:75858", source="Orphanet:75858/e"}
-is_a: MONDO:0016565 {source="Orphanet:75858"} ! syndromic genetic obesity
-is_a: MONDO:0019118 {source="MONDO:Redundant", source="Orphanet:75858"} ! inherited retinal dystrophy
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+relationship: excluded_subClassOf MONDO:0019118 {source="Orphanet:75858"} ! inherited retinal dystrophy
```

Comment:
instead of deleting the subclass assertion, we just exclude, so we can retain the history/provenance that it came from Orphanet

here is the documentation about it: https://mondo.readthedocs.io/en/latest/editors-guide/g-logical-axioms/#excluded-subclassof